ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/blockhash/PragueBlockHashProcessor.java

```diff
@@ -55,7 +60,12 @@ public Void process(final BlockProcessingContext context) {
         new SystemCallProcessor(context.getProtocolSpec().getTransactionProcessor());
 
     Bytes inputData = context.getBlockHeader().getParentHash();
-    processor.process(historyStorageAddress, context, inputData);
+    try {
+      processor.process(historyStorageAddress, context, inputData);
+    } catch (InvalidSystemCallAddressException e) {
+      // According to EIP-2935, the system call should fail silently if the address does not exist
```

Comment:
I think this comment is slightly misleading. As I understand the code this exception could also be thrown if no code exists at the address

Reply:
fixed